recipes/treeqmc/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set name = "TREEQMC" %}
```

Comment:
Should it be `TREE-QMC` ?

Reply:
The package name has been updated and now it is tree-qmc.